apps/tx-builder/src/components/modals/ImplementationABIDialog.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { Text, Button, GenericModal, EthHashInfo } from '@gnosis.pm/safe-react-components'
+import Box from '@material-ui/core/Box'
+import styled from 'styled-components'
+import { confirmable, ConfirmationProps } from '../utils/Confirmable'
+
+type Props = {
+  networkPrefix: string
+  implementationAddress: string
+  blockExplorerLink: string
+} & ConfirmationProps
+
+const ImplementationABIDialog: React.FC<Props> = ({
+  networkPrefix,
+  implementationAddress,
+  blockExplorerLink,
+  proceed,
+}) => {
+  return (
+    <GenericModal
```

Comment:
If you close the modal using the Cross icon, the input field gets stuck in an infinite loader state:

![Captura de pantalla 2022-07-18 a las 15 00 58](https://user-images.githubusercontent.com/26763673/179517095-8ee89ae1-be44-4cce-b058-f73ca03ce064.png)
![Captura de pantalla 2022-07-18 a las 15 01 04](https://user-images.githubusercontent.com/26763673/179517106-c9ac8799-2f04-42fe-9b24-6d03d06283ad.png)

Reply:
Fixed, thanks, senor! Was handling the case with empty abi returned from the service incorrectly